doc/swagger.yaml

```diff
@@ -1931,7 +1950,7 @@ paths:
         name: request
         required: true
         schema:
-          $ref: '#/definitions/pkg_server_router.StoreKeyRequest'
+          $ref: '#/definitions/github.com_tbd54566975_ssi-service_pkg_server_router.StoreKeyRequest'
```

Comment:
I'm not sure why this happens. Curious if it's something to do with my version of swaggo?

Reply:
I don't think it really matters as long as it gets rendered properly